android/src/main/java/cl/json/social/WhatsAppBusinessShare.java

```diff
@@ -19,6 +20,24 @@ public WhatsAppBusinessShare(ReactApplicationContext reactContext) {
     @Override
     public void open(ReadableMap options) throws ActivityNotFoundException {
         super.open(options);
+        
+        if (options.hasKey("whatsAppNumber")) {
+            // create an empty conversation in case it's not on contacts
+            this.getIntent().setComponent(new ComponentName(PACKAGE, "com.whatsapp.Conversation")); 
+            this.openIntentChooser();
+
+
+            // leave room for the conversation to be created
+            try {
+                Thread.sleep(300);   
```

Comment:
What about setting this `300` to a constant? It would be much easier to change/find what this number does.